### DISCUSSION TITLE
My piano have 76 key

### DISCUSSION THREAD

**Discussion by maisonnavejul** [onlaj/Piano-LED-Visualizer#283] [Q&A]
Hey ,
What i can change for my programme work in 76 key and not in 88?

**Comment by onlaj** [comment]
Hello.
Just change it in settings, on screen or web interface.

**Reply by maisonnavejul** [reply]
in led Count i change to 76 but nothing change

**Reply by onlaj** [reply]
What changes do you expect? Are leds aligned to keys?

**Reply by maisonnavejul** [reply]
i have one led for one key, and i want to use the programme with piano 76 key.
The programme start here like a piano 88 key :
![image](https://user-images.githubusercontent.com/92145165/143769976-78217202-3095-447c-9cdb-f7cf796c7551.png)
but i want the programme start for et piano 76 key, like that :
![image](https://user-images.githubusercontent.com/92145165/143770003-fed1adf0-14fc-4ae3-a0a1-bcb2daaf576a.png)

**Reply by onlaj** [reply]
Use "shift" option then.

> i have one led for one key

If by that you mean that your led strip has less than 144 leds/m it won't work, there is no support for other density.

**Reply by maisonnavejul** [reply]
okay thanks, i will try to chane that

**Reply by maisonnavejul** [reply]
change*
